base_exception/models/base_exception.py

```diff
@@ -202,6 +218,16 @@ def _compute_main_error(self):
             else:
                 rec.main_exception_id = False
 
+    @api.depends('exception_ids', 'ignore_exception')
+    def _compute_exception_ids_formatted(self):
+        for rec in self:
+            if rec.ignore_exception or not rec.exception_ids:
+                rec.exception_ids_formatted = False
```

Comment:
IMHO this line is not necessary as before compute, values are void

Reply:
Absolutely right. Just removed them